bskyweb/static/manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "short_name": "Bluesky",
+  "name": "Bluesky",
+  "icons": [
+    {
+      "src": "/static/apple-touch-icon.png",
+      "type": "image/png",
+      "sizes": "1024x1024"
+    }
+  ],
+  "start_url": "/",
+  "display": "standalone",
```

```suggestion
  "display": "standalone",
  "display_override": ["tabbed"],
```

Since this is a social network app and numberous links are likely to be clicked, my opinion is that we should open with tabs too.

https://developer.chrome.com/docs/capabilities/tabbed-application-mode#using_tabbed_application_mode

---

I'm not sure this is a good idea given this property is experimental:

https://developer.mozilla.org/en-US/docs/Web/Manifest/display_override